contracts/library/LoanLib.sol

```diff
@@ -23,27 +23,39 @@ library LoanLib {
     using SafeMath  for uint256;
     using SafeERC20 for IERC20;
 
-    enum State { Ready, Active, Matured, Expired, Liquidated }
-
     address public constant UNISWAP_ROUTER = 0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D;
```

Comment:
Yeah uniswap uses CREATE2 so their addresses are the same across all networks: https://rinkeby.etherscan.io/address/0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D#code

Reply:
As long as its constant or immuable, it's all the same. I do agree consistency is nice, and it would be a cosmetic change at this point. Albeit a welcome one.